src/main/resources/assets/controlify/lang/fr_fr.json

```diff
@@ -0,0 +1,393 @@
+  "controlify.gui.allow_vibrations.tooltip": "Si la manette doit vibrer lorsque vous effectuez certaines actions.",
+  "controlify.gui.allow_vibrations.not_available": "Votre manette ne prend pas en charge les vibrations ou les pilotes de vibration ne sont pas chargés."
+  "controlify.gui.group.gyro": "Gyroscope",
+  "controlify.gui.group.gyro.tooltip": "Ajuster la façon dont Controlify traite le gyroscope intégré de votre manette.\nUn gyroscope détermine la rotation de la manette.",
+  "controlify.gui.group.gyro.no_gyro.tooltip": "Cette manette ne prend pas en charge le gyroscope. Vous devez posséder une manette DualSense™ ou une autre manette compatible pour utiliser cette fonctionnalité.",
+  "controlify.gui.group.gyro.no_gyro_steamdeck.tooltip": "Le gyroscope du Steam Deck ne fonctionne actuellement pas avec Controlify. Pour l'instant, utilisez Steam Input pour émuler le gyroscope et l'utiliser comme une souris.",
+  "controlify.gui.gyro_look_sensitivity": "Sensibilité de la vision gyroscopique",
+  "controlify.gui.gyro_look_sensitivity.tooltip": "Le pourcentage de rotation réelle de votre manette utilisé comme entrée de vision. Par exemple, une rotation de votre manette de 90 degrés, déplacera la caméra de 90 degrés, à 100%.\nLe tangage de votre manette est utilisé pour le haut/bas et le roulis et le lacet sont utilisés pour la gauche/droite.\nCette sensibilité est complètement indépendante des paramètres de sensibilité horizontale et verticale habituels.",
+  "controlify.gui.gyro_behaviour": "Comportement du gyroscope",
+  "controlify.gui.gyro_behaviour.tooltip": "Comment l'entrée du gyroscope doit être interprétée comme entrée de vision.",
+  "controlify.gui.gyro_behaviour.absolute": "Absolu",
+  "controlify.gui.gyro_behaviour.absolute.tooltip": "Absolu : Équivalent à déplacer la souris dans une direction.",
+  "controlify.gui.gyro_behaviour.relative": "Relatif",
+  "controlify.gui.gyro_behaviour.relative.tooltip": "Relatif : Équivalent à déplacer un joystick dans une direction.",
+  "controlify.gui.gyro_invert_x": "Inverser l'axe X",
+  "controlify.gui.gyro_invert_x.tooltip": "Inverser la rotation gauche/droite de la direction de la vision gyroscopique.",
+  "controlify.gui.gyro_invert_y": "Inverser l'axe Y",
+  "controlify.gui.gyro_invert_y.tooltip": "Inverser la rotation haut/bas de la direction de la vision gyroscopique.",
+  "controlify.gui.gyro_requires_button": "Nécessite un bouton",
+  "controlify.gui.gyro_requires_button.tooltip": "Si le gyroscope doit être utilisé uniquement lorsque la commande gyroscopique est enfoncée (défilement jusqu'aux commandes).",
+  "controlify.gui.flick_stick": "Flick Stick",
+  "controlify.gui.flick_stick.tooltip": "Modifie le comportement des commandes de regarder en haut/bas/gauche/droite pour faire tourner la direction de vision de 90 degrés dans la direction correspondante lorsqu'appuyé. Ceci devrait être combiné avec la vision gyroscopique pour obtenir une visée précise et rapide.",
```

Comment:
Difficile à comprendre, mais le texte initial n’est pas très clair non plus.  Je propose :
"Modifie le comportement des boutons vue haut/bas/gauche/droite pour faire tourner la vue à 90 degrés dans la direction appuyée."

Reply:
Sorry, French is not my main language.
I trust you for this one.
Modified.